modules/wri_seo/wri_seo.info.yml

```diff
@@ -23,7 +23,6 @@ dependencies:
   - xmlsitemap
 config_devel:
   install:
-    - metatag.settings
```

Comment:
I assume this is fine, but I wanted to flag it because it wasn't immediately obvious why it's being removed.

Reply:
This part of the info file lists out what config files are in the install folder if you're using the `config_devel` module. It helps you then quickly export any changed config from your site's `config` folder into the correct module using drush. Since I moved the `metatag.settings.yml` file out of this module and into the profile's `config/install` folder, I'm just keeping  this bit of housekeeping consistent!